### DISCUSSION TITLE
Speed and performance

### DISCUSSION THREAD

**Discussion by AppLetico** [callstack/react-native-builder-bob#632] [Q&A]
Is there anyway to make bob build run faster?  Currently it takes about 15 seconds to compile.   Is there a way to enabling caching of untouched files to skip them, or run things in parallel?

Thanks

nodemon] starting `yarn prepare`
ℹ Building target commonjs
ℹ Cleaning up previous build at lib/commonjs
ℹ Building target module
ℹ Cleaning up previous build at lib/module
ℹ Compiling 196 files in src with babel
ℹ Compiling 196 files in src with babel
✔ Wrote files to lib/commonjs
✔ Wrote files to lib/module

**Comment by satya164** [comment]
They already run in parallel.
Given that they only run during publish, potential issues due to outdated caching aren't worth it.
